.luacheckrc

```diff
@@ -46,4 +46,7 @@ globals = {
     "client"
 }
 
+-- Enable cache (uses .luacheckcache relative to this rc file).
+cache = true
```

Comment:
Sorry, I seem to be wrong. It seems like the `cache` option silently does nothing if `lfs` is not available.